src/chttpd/src/chttpd_rewrite.erl

```diff
@@ -442,7 +442,7 @@ path_to_list([<<>>|R], Acc, DotDotCount) ->
 path_to_list([<<"*">>|R], Acc, DotDotCount) ->
     path_to_list(R, [?MATCH_ALL|Acc], DotDotCount);
 path_to_list([<<"..">>|R], Acc, DotDotCount) when DotDotCount == 2 ->
-    case config:get("httpd", "secure_rewrites", "true") of
+    case chttpd_util:get_chttpd_config(secure_rewrites, "true") of
```

Comment:
Let use `get_chttpd_config_boolean(secure_rewrites, true)` since we a boolean getter function now. Then result should be `false` or `true`.